### PR TITLE
osv-scanner: add page

### PR DESCRIPTION
**Pull request by fnfontana**
The Open Source Vulnerability (OSV) Database is a distributed vulnerability database for open source ecosystems.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/osv-scanner.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`osv-scanner -L {{path/to/lockfile}}`
+
+- Scan an SBOM file:
+
+`osv-scanner -S {{path/to/sbom_file}}`
+
+- Scan multiple directories recursively:
+
+`osv-scanner -r {{directory1 directory2...}}`
```
Comment (blueskyson) [inline pages/common/osv-scanner.md:20]:
```suggestion
`osv-scanner -r {{directory1 directory2 ...}}`
```

File: pages/common/osv-scanner.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Skip scanning git repositories:
+
+`osv-scanner --skip-git`
```
Comment (blueskyson) [inline pages/common/osv-scanner.md:24]: Should this option accompany by `-r` or `-D`?
```suggestion
`osv-scanner --skip-git` {{-r|-D}} {{target}}
```

File: pages/common/osv-scanner.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Output result in JSON format:
+
+`osv-scanner --json`
```
Comment (blueskyson) [inline pages/common/osv-scanner.md:28]: Should this option accompany by any of the scan options?
```suggestion
`osv-scanner --json {{-D|-L|-S|-r}} {{target}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @fnfontana for your contribution.

**Pull request merged**